plugins/Rockets/jsonPropertyMap.h

```diff
@@ -118,12 +201,19 @@ void _addPropertySchema(const PropertyMap::Property& prop,
     {
         jsonSchema.SetObject();
         jsonSchema.AddMember(StringRef("type"), StringRef("string"), allocator);
+        jsonSchema.AddMember(StringRef("title"), StringRef(prop.label.c_str()),
+                             allocator);
+        jsonSchema.AddMember(StringRef("readOnly"), prop.readOnly(), allocator);
+
+        // Specialized enum code
+        auto value = prop.get<int32_t>();
```

Comment:
so enums must still be `int32_t` :stuck_out_tongue: So the code in PropertyMap.h shall be put back.

Reply:
Not exactly. The case for string enums is below in ```_addPropertySchema<std::string>```.